### DISCUSSION TITLE
Testing jsonConfig.* modules

### DISCUSSION THREAD

**Discussion by kpodsiad** [scalacenter/bloop#1643] [Q&A] [answered]
As far I can see, [CI](https://github.com/scalacenter/bloop/blob/master/.github/workflows/ci.yml) isn't testing any of `jsonConfig` modules. Is this correct? There are some tests in these modules.

**Comment by tgodzik** [comment, marked as answer]
I don't see any tests there, which ones do you mean?

**Reply by kpodsiad** [reply]
`jsonConfig` modules have `.in(file("config"))` so for example https://github.com/scalacenter/bloop/blob/master/config/src/test/scala/bloop/config/utils/BaseConfigSuite.scala

**Reply by tgodzik** [reply]
This is just a trait needed for integration tests, so we can move it somewhere else.

**Reply by kpodsiad** [reply]
Right, I didn't spot that ;) But there is also [ConfigCodeSpec](https://github.com/scalacenter/bloop/blob/053cdea7a45c4395f3835b4913d83a73a736e7c7/config/src/test/scala/bloop/config/ConfigCodecsSpec.scala) which is a class. Probably we should move it to the other module or start running this on CI.

**Reply by tgodzik** [reply]
We can move it to frontend-test, though the tests have been ignored for a while, so they might be failing.

**Comment by kpodsiad** [comment]
Also, why there is a full combination of  `jsonConfig` + {210, 211, 212, 213} + {JVM, JS} needed?
